src/components/PostTable/Header/Header.styles.js

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+
+const Row = styled.div`
+  display: flex;
+  width: ${({ theme }) => theme.postTable.width};
+  border-top: 1px solid ${({ theme }) => theme.postTable.borderColor};
+  border-right: 1px solid ${({ theme }) => theme.postTable.borderColor};
+  color: ${({ theme }) => theme.postTable.headerColor};
```

Comment:
The font colors don't seem to be correct. Actually I guess the global font-color is not correct